ddp-workspace/projects/ddp-sdk/src/lib/components/login/auth0-code-callback.component.ts

```diff
@@ -64,7 +64,12 @@ export class Auth0CodeCallbackComponent implements OnInit, OnDestroy {
                 this.windowRef.nativeWindow.location.href = nextUrl;
             });
         } else {
-            throw new Error('No auth code present in url.');
+            const error = this.route.snapshot.queryParams['error'];
+            const errorDescription = this.route.snapshot.queryParams['error_description'];
+            this.log.logError(this.LOG_SOURCE, 'No auth code present in url');
+            this.log.logError(this.LOG_SOURCE, `Error: ${error}`);
+            this.log.logError(this.LOG_SOURCE, `Error Description: ${errorDescription}`);
+            this.adapter.logout();
```

Comment:
Since this component is in the SDK and not toolkit, I don't think we have a general SDK "error page", so the logout will currently redirect to homepage. That might be confusing but I think its okay for local development.

Reply:
We have error page URL in the SDK Configuration service.
```suggestion
            this.adapter.logout(this.configuration.errorPageUrl);
```

Reply:
Sweet, thanks!